utils/timeUtils.ts

```diff
@@ -50,6 +50,26 @@ export function formatPostedAt(isoDate: string): string {
   return result;
 }
 
+export function dateToAge( dob: Date ) : number | null {
```

Comment:
This function works and is correct but not what we need - birthday is stored as an isodate (string) after its in the database, so we need to be able to take that string and convert it to an age. Right now its converting a Date object

For now just remove the timeUtils changes since theyre separate from the rest of the PR, and read the ticket on Jira for more info